test/extended/builds/pipeline_jenkins_e2e.go

```diff
@@ -194,466 +194,469 @@ var _ = g.Describe("[sig-devex][Feature:Jenkins][Slow] Jenkins repos e2e openshi
 
 	// these tests are isolated so that PR testing the jenkins-client-plugin can execute the extended
 	// tests with a ginkgo focus that runs only the tests within this ginkgo context
-	g.Context("Sync plugin tests", func() {
-
-		g.It("using the persistent template", func() {
-			defer cleanup(jenkinsPersistentTemplatePath)
-			setupJenkins(jenkinsPersistentTemplatePath)
-			// additionally ensure that the build works in a memory constrained
-			// environment
-			_, err := oc.AdminKubeClient().CoreV1().LimitRanges(oc.Namespace()).Create(context.Background(), &corev1.LimitRange{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: "limitrange",
-				},
-				Spec: corev1.LimitRangeSpec{
-					Limits: []corev1.LimitRangeItem{
-						{
-							Type: corev1.LimitTypeContainer,
-							Default: corev1.ResourceList{
-								corev1.ResourceMemory: resource.MustParse("512Mi"),
+	g.Context("", func() {
+
+		g.Describe("Sync plugin tests", func() {
```

Comment:
I don't fully understand why we have to nest `Describe` underneath `Context` - especially since the Ginkgo docs suggest Describe -> Context is the preferred nesting structure [1]. Please add a comment documenting the timing issue, and why nesting Describe underneath Context addresses this issue.

[1] https://onsi.github.io/ginkgo/#organizing-specs-with-containers-describe-and-context

Reply:
some context here in the interim ... let's talk in scrum, and then decide if/where to comment 

- the pipeline tests used to be structured like this ... there is a long history around slow tests and memory concerns throughout the releases of openshift
- it you look at both build e2e's and other e2e's, you'll actually see a similar structure ... I can present more clearly during scrum ... as we will be making this look like most other e2e's now, not sure if a comment in code makes sense, but we'll see
- the wonderful vagaries around ginkgo and how it integrates with k8s e2e start at https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/test/e2e/e2e.go#L76-L84 ... it was in debugging this path and our `pipeline_*.go` creation of their oc CLI .... `exutil.NewCLI("jenkins-pipeline") that all this started

Reply:
we discussed this in scrum ... the net: no new comment here, but some investigation in finding and/or doc'ing if need be best practices around this concurrency and make sure suites conform to that best practice will follow after this PR